js/src/Ice/AsyncResult.js

```diff
@@ -91,34 +90,6 @@ export class AsyncResult extends Promise {
         this._cancellationHandler = handler;
     }
 
-    startReadParams() {
```

Comment:
The exact same methods are provided by the derived class `OutgoingAsync`. AsyncResult doesn't have an InputStream doesn't make sense to define them here.